bitnami/prometheus/Chart.yaml

```
@@ -39,4 +39,4 @@ sources:
 - https://github.com/bitnami/containers/tree/main/bitnami/prometheus
 - https://github.com/prometheus/prometheus
 - https://github.com/prometheus-community/helm-charts
-version: 2.0.8
\ No newline at end of file
+version: 2.0.9
```

Please bump minor version instead given you're adding new parameters:
```suggestion
2.1.0
```